python/jibe_controller.py

```diff
@@ -0,0 +1,19 @@
+# from heading_controller import HeadingController
+# import sailbot_constants
+# import math
+
+
+class JibeController():
```

Comment:
I think Joel's jibe controller is merged so we can use the actual implementation

Reply:
It is removed